python/tk_desktop/project_model.py

```diff
@@ -214,6 +215,22 @@ class SgProjectModel(ShotgunModel):
     thumbnail_updated = QtCore.Signal(QtGui.QStandardItem)
     project_launched = QtCore.Signal()
 
+    _supports_project_templates = None
+
+    @staticmethod
```

Comment:
Not sure if the difference is intentional (to lock subclasses from being able to change this behavior) but I'd expect this to be

```python
@classmethod
def supports_project_templates(cls, connection):
...
```

And then replace the ```SgProjectModel``` explicit symbols in the definition with ```cls```.
Is there a reason to make it static as opposed to class?